docs/content/preview/yugabyte-voyager/reference/data-migration/import-data.md

```diff
@@ -232,8 +232,8 @@ The valid *arguments* for import data to source-replica are described in the fol
 | -h, --help | Command line help for import data to source-replica. |
 | --oracle-home <path> | Path to set $ORACLE_HOME environment variable. `tnsnames.ora` is found in `$ORACLE_HOME/network/admin`. Oracle migrations only.|
 | [--oracle-tns-alias](../../yb-voyager-cli/#ssl-connectivity) <alias> | TNS (Transparent Network Substrate) alias configured to establish a secure connection with the server. Oracle migrations only. |
-| --parallel-jobs <connectionCount> | The number of parallel batches issued to the source-replica database. <br> Default: 1 |
-| --start-clean | Starts a fresh import with data files present in the `data` directory.<br>If there's any non-empty table on the target YugabyteDB database, you get a prompt whether to continue the import without truncating those tables; if you go ahead without truncating, then yb-voyager starts ingesting the data present in the data files with upsert mode.<br> **Note** that for cases where a table doesn't have a primary key, it may lead to insertion of duplicate data. In that case, you can avoid the duplication by excluding the table from the `--exclude-table-list`, or truncating those tables manually before using the `start-clean` flag. <br>Default: false<br> Accepted parameters: true, false, yes, no, 0, 1 |
+| --parallel-jobs <connectionCount> | The number of parallel batches issued to the source-replica database. <br> Default: Oracle(16) and PostgreSQL (If yb-voyager can determine the total number of cores N, then use N/2 as parallel jobs, else it falls back to 8) |
```

Comment:
```suggestion
| --parallel-jobs <connectionCount> | The number of parallel batches issued to the source-replica database. <br> Default: 16 (Oracle); or, if yb-voyager can determine the total number of cores N, then N/2, otherwise 8 (PostgreSQL) |
```